events/ri/build.gradle.kts

```diff
@@ -18,6 +18,11 @@ import org.apache.tools.ant.taskdefs.condition.Os
 
 plugins {
   alias(libs.plugins.quarkus)
+    .version(
+      libs.plugins.quarkus.asProvider().map {
+        System.getProperty("quarkus.custom.version", it.version.requiredVersion)
+      }
+    )
```

Comment:
This mechanism is sadly the only option I could come up with to align the Quarkus plugin version with a requested custom Quarkus version.

Within a `plugins {}`, it's impossible to use `project` and it's impossible to access anything in `build-logic` (Gradle just complains with a `ClassNotFoundException).